frontend-react/src/views/Drive/components/DriveForm/mockItems.js

```diff
@@ -0,0 +1,45 @@
+// values to generate MenuItems in select fields. Will be converted to redux slice.
```

Comment:
This file is redundant, cars, passengers and projects are fetched from backend and real data are used in the form

Reply:
I can't recall creating this file, removed nonetheless